components/brave_vpn/resources/panel/container.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react'
+import SelectRegion from './components/select-region'
+import MainPanel from './components/main-panel'
+import apiProxy from './vpn_panel_api_proxy.js'
+import { ConnectionState } from './types/connection_state'
+
+function Main () {
+  const [isOn, setOn] = React.useState(false)
+  const [isSelectingRegion, setSelectingRegion] = React.useState(false)
+  const [hasError, setHasError] = React.useState(false)
+  const [status, setStatus] = React.useState<ConnectionState>(ConnectionState.DISCONNECTED)
+
```

Comment:
@petemill ideally all of the states should be moved to react context. Once I test and understand how auth and fetching region list would work I can solidify this.